src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -66,30 +55,41 @@ $active-poster-shadows: 0 10px 20px 0 var(--primary--under-50a);
   &__categories {
     display: flex;
     flex-wrap: wrap;
+    margin-top: $spacing--md;
   }
 
   &__authorBox {
     display: flex;
     align-items: center;
-    margin-top: auto;
-  }
-
-  &__avatar {
-    margin-right: auto;
-  }
-
-  &__author {
-    margin-left: $spacing--sm;
     font-size: $font-size--sm;
-    font-weight: $font-weight--500;
-    @include line-clamp-with-overflow(5);
   }
 
-  &__contactEmail {
-    font-size: $font-size--sm;
+  &__thumbnailContainer {
```

Comment:
How did camel case get into this? Copy from legacy code? Could use a `@debt` comment

Reply:
Resolved in 461139c50